src/Components/BehaviorsModal/Internal/Widgets/DataHistoryWidget/DataHistoryWidget.tsx

```diff
@@ -47,8 +47,9 @@ import {
     IDataHistoryWidgetStyleProps,
     IDataHistoryWidgetStyles
 } from './DataHistoryWidget.types';
+import { DataHistoryWidgetErrorHandling } from './Internal/DataHistoryWidgetErrorHandling';
 
-const getClassNames = classNamesFunction<
```

Comment:
can we add a story or two to cover the different error cases?

Reply:
added stories for each surfaced error under Components/Behaviors Modal/Data History